SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialPackageMapClient.cpp

```diff
@@ -507,6 +507,12 @@ FNetworkGUID FSpatialNetGUIDCache::AssignNewEntityActorNetGUID(AActor* Actor, Wo
 		SlowCheckMapsConsistency(Subobject);
 	}
 
+	// If the actor is initially dormant, we must register it to our dormancy map.
+	if (Actor->NetDormancy == ENetDormancy::DORM_Initial)
+	{
+		SpatialNetDriver->RegisterDormantEntityId(EntityId);
+	}
```

Comment:
Hmm, I am a bit worried that maybe there should be a different fix? But I don't know enough about dormancy.
Not sure if an actor channel should never be created, or it should be created but immediately destroyed...
Would also be nice to know when this behaviour broke, as it used to work... a long time ago.